canonical-webteam/deployment-configs
# Add environement status in configMap

# Summary

- Add evnvironlent var to a new config-map so that projects can get the environment where they are hosted (production/staging)

# QA

- `./qa-deploy --tag latest --production snapcraft.io`
- Create configmap.yaml : 
```yaml
apiVersion: v1
data:
  ENVIRONMENT: "devel"
kind: ConfigMap
metadata:
  name: global-config
```
- `kubectl apply -f configmap.yaml`
- Make sure s.io uses the environment var: `ENVIRONMENT`
